pyvista/_vtk.py

```diff
@@ -543,6 +543,12 @@ def lazy_vtkPOpenFOAMReader():
         reader.SetController(vtkDummyController())
         return reader
 
+    def lazy_vtkXdmfReader():
+        """Lazy import of the vtkXdmfReader."""
+        from vtkmodules.vtkIOXdmf3 import vtkXdmfReader
```

Comment:
```suggestion
        from vtkmodules.vtkIOXdmf2 import vtkXdmfReader
```

Reply:
The confusing thing is : there _is_ a XDMF3 reader in vtk. See [here](https://vtk.org/doc/nightly/html/classvtkXdmf3Reader.html)

I just don't know how to import it in python...

Reply:
It seems that there is no XDMF3 under `vtkmodules`. I will add XDMF2 to this Pull Request.
--------------------------------------------------------------------------------
  Date: Wed Oct 26 09:15:25 2022 JST

                OS : Linux
            CPU(s) : 8
           Machine : x86_64
      Architecture : 64bit
               RAM : 31.3 GiB
       Environment : IPython
       File system : ext4
        GPU Vendor : Intel
      GPU Renderer : Mesa Intel(R) HD Graphics 4600 (HSW GT2)
       GPU Version : 4.6 (Core Profile) Mesa 22.0.5

  Python 3.10.6 (main, Aug 10 2022, 11:40:04) [GCC 11.3.0]

           pyvista : 0.37.dev0
               vtk : 9.2.2
             numpy : 1.23.3
           imageio : 2.22.0
           appdirs : 1.4.4
            scooby : 0.5.12
        matplotlib : 3.6.0
           IPython : 8.5.0
          colorcet : 3.0.0
           cmocean : 2.0
        ipyvtklink : 0.2.2
             scipy : 1.9.1
              tqdm : 4.64.1
            meshio : 5.3.4
        jupyterlab : 3.4.7
         pythreejs : Version unknown
--------------------------------------------------------------------------------

Reply:
Thanks. Great idea to post a question in vtk community.

Reply:
According to its answer, it seems that XDMF2 can provide stable functionality. Are there any particular advantages of XDMF3?

Reply:
I don't think so no!